virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -1354,77 +1335,118 @@ You can configure a DataSource on the DataSource details page.
 |Description
 
 |*Details* tab
-|Configure a DataSource by editing a form.
+|Configure the data source of a bootable volume by editing a form.
 
 |*YAML* tab
-|Configure a DataSource by editing a YAML configuration file.
+|Configure the data source of a bootable volume by editing a YAML configuration file.
 
 |*Actions* menu
 |Select *Edit labels*, *Edit annotations*, *Delete*, or *Manage source*.
 
-|Name
-|DataSource name
+*Manage source* is disabled for Red Hat data sources.
+|====
+=====
 
-|Namespace
-|DataSource namespace
+[id="bootablevolume-details-details_virt-web-console-overview"]
+==== Details tab
 
-|DataImportCron
-|DataSource DataImportCron
+You configure the data source of the bootable volume by editing a form on the *Details* tab.
 
-|Labels
+.*Details* tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|*Name*
+|Data source name.
+
+|*Namespace*
+|Data source namespace. The default project is *openshift-virtualization-os-images*.
+
+|*DataImportCron*
+|The `DataImportCron` polls disk images and imports them as persistent volume claims.
+
+|*Labels*
 |Click the edit icon to edit the labels.
 
-|Annotations
+|*Annotations*
 |Click the edit icon to edit the annotations.
 
-|Conditions
-|Displays the status conditions of the DataSource.
+|*Source*
+|Source of data source.
+
+|*Created at*.
+|Data source creation date.
 
-|Created at
-|DataSource creation date
+|*Owner*
+|Data source owner.
 
-|Owner
-|DataSource owner
+|*Conditions* table
+|Displays the status of the data source.
 |====
 =====
 
-[id="ui-migrationpolicies-page"]
-== MigrationPolicies page
+[id="bootablevolume-details-yaml_virt-web-console-overview"]
+==== YAML tab
 
-You can manage MigrationPolicies for your workloads on the MigrationPolicies page.
+You configure the data source of the bootable volume by editing the YAML file on the *YAML* tab.
 
-.MigrationPolicies page
+.*YAML* tab
 [%collapsible]
 =====
 [cols="1,3a", options="header"]
 |====
 |Element
 |Description
 
-|*Create MigrationPolicy -> With form*
-|Create a MigrationPolicy by entering configurations and labels in a form.
+|*Save* button
+|Save changes to the YAML file.
+
+|*Reload* button
+|Discard your changes and reload the YAML file.
+
+|*Cancel* button
+|Exit the *YAML* tab.
+
+|*Download* button
+|Download the YAML file to your local machine.
+|====
+=====
+
+[id="migrationpolicies-page_virt-web-console-overview"]
+== MigrationPolicies page
```

Comment:
I think this instance should be "Migration policies page"

Reply:
But it is called "MigrationPolicies" in the UI, so I do not think we can call it "Migration policies".